.github/workflows/ci.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           sudo npm install -g prettier
           pip install -U pip wheel setuptools
           pip install -U -r requirements-test.txt
-          pip install -e .[saml,openvpn_status]
+          pip install -e .[saml,openvpn_status,selenium]
```

Comment:
The selenium dependency has been centralized in openwisp-utils. We don't need this here.